lib/src/sdk_env.dart

```diff
@@ -275,14 +275,16 @@ class ToolEnvironment {
     bool validateLinks: true,
     bool linkToRemote: false,
     Duration timeout,
+    List<String> excludedLibs,
   }) async {
     ProcessResult pr;
     final args = [
       '--output',
       outputDir,
-      '--exclude',
-      dartdocExcludedLibraries.join(','),
```

Comment:
Is `dartdocExcludedLibraries` still in use? Otherwise we could remove it too.

Reply:
good point, excluded.